vshaxe-build.json

```diff
@@ -31,7 +31,6 @@
 				},
 				"macros": [
 					"haxeLanguageServer.Init.run()",
-					"Safety.safeNavigation('haxeLanguageServer')",
```

Comment:
`safety` lib can be removed now I think? If not, what is still being used?

Reply:
Its used for `.sure()` calls with `using Safety` maybe in ten places. It can be replaced with `value ?? throw 'some error'` if needed.

Reply:
Right, makes sense.